tests/https/test_unit_endpoints.py

```diff
@@ -59,3 +59,12 @@ def test_endpoint_has_correct_role_template_string(self):
                                                              kind='rolekind',
                                                              identifier='roleid')
         self.assertEqual(endpoint, 'http://host/roles/myacct/rolekind/roleid')
+
+    def test_endpoint_has_correct_check_privilege_template_string(self):
+        endpoint = ConjurEndpoint.PRIVILEGE.value.format(url='http://host',
+                                                             account='myacct',
+                                                             kind='resourcekind',
+                                                             identifier='resourceid',
+                                                             role='roleid',
+                                                             privilege='respriv')
+        self.assertEqual(endpoint, 'http://host/resources/myacct/resourcekind/resourceid?check=true&role=roleid&privilege=respriv')
```

Comment:
added check to verify that the template matches the URI in the docs.

output snippet from unit testing:

`test_endpoint_has_correct_check_privilege_template_string (tests.https.test_unit_endpoints.EndpointsTest) ... ok`